### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by gislenius**

### CODE REVIEW

**Comment by wcandillon** [comment]
@gislenius Can you link the documentation from the readme?

File: documentation/rest/chapter-1-introduction.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+##About the sample REST links (for developers)
+For pedagogical reasons, links are only shown with the relevant parameters. When you create your own links, however, you need to add a token parameter. On this sample DB, the token is publicly available and is 54eb0587-1aee-44a5-bb56-627f6ef43da6.
+
+Also, the HTTP requests need to to sent as POST (although they will very soon be available as GET as well). For convenience, you can add &_method=POST if you are testing the links in your favorite browser. If you are using curl, add -X POST, etc.
+
+So, when you see in this tutorial:
+
+```REST
+http://edinet.28.io/v1/_queries/public/api/entities.jq?ticker=7751
```
Comment (wcandillon) [inline documentation/rest/chapter-1-introduction.md]: @gislenius no token?

File: documentation/rest/chapter-1-introduction.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+```
+
+You actually need to understand it as being really:
+
+```REST
+http://edinet.28.io/v1/_queries/public/api/entities.jq?ticker=7751&token=54eb0587-1aee-44a5-bb56-627f6ef43da6&_method=POST
```
Comment (wcandillon) [inline documentation/rest/chapter-1-introduction.md]: @gislenius This doesn't seem to be the master token.

**Comment by wcandillon** [comment]
@gislenius Awesome stuff :+1:. I need to figure how we can ship our documentation in a coherent package with the API documentation.

File: documentation/rest/chapter-1-introduction.md (hunk 1 above)
Comment (gislenius) [inline documentation/rest/chapter-1-introduction.md]: It's supposed to be added automatically by your automation part. Of course we can change the input format if it makes things easier, like { ticker: 7751 }. But the end reader would see the URL.

**Comment by gislenius** [comment]
I address your comments.

**Comment by wcandillon** [comment]
@gislenius Please have a look at https://www.gitbook.com to make sure everything is compliant. For instance, you probably need a `README.md` and `SUMMARY.md` file in the documentation folder.

**Comment by gislenius** [comment]
@wcandillon that's done!

**Comment by wcandillon** [comment]
@gislenius Can you look at #730

**Comment by wcandillon** [comment]
@gislenius the deployment at http://edinet.xbrl.io-blog.s3-website-us-east-1.amazonaws.com/documentation/ seems to show dead links.

**Pull request merged**